static/js/controllers/reports.js

```diff
@@ -250,17 +336,18 @@ var _ = require('underscore'),
             _query();
           }
         }
-
       };
 
       $scope.$on('ClearSelected', function() {
```

Comment:
No. $scope.clearSelected emits this event so child controllers can clean up their context, so doing that would cause an infinite loop...